vera/codegen/api.py

```
@@ -2078,6 +2083,64 @@ def host_inference_complete(
+                "Cannot allocate String argument: module has no 'alloc' export"
+            )
+            encoded = s.encode("utf-8")
+            ptr = alloc_export(store, len(encoded))
+            assert isinstance(ptr, int)
+            buf = memory_export.data_ptr(store)
+            for i, b in enumerate(encoded):
+                buf[ptr + i] = b
+            return ptr, len(encoded)
+
+        parsed: list[int | float] = []
+        for raw, wasm_type in zip(raw_args, vera_params):
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)